bom/templates/bom/part-info.html

```diff
@@ -323,6 +325,8 @@ <h4 class="printer-show">Sourcing</h4>
                                     <td id="sourcing-mp-{{ manufacturer_part.id }}-lt">{{ manufacturer_part.optimal_seller.lead_time_days }}</td>
                                     <td id="sourcing-mp-{{ manufacturer_part.id }}-nre">{{ manufacturer_part.optimal_seller.nre_cost }}</td>
                                     <td id="sourcing-mp-{{ manufacturer_part.id }}-ncnr">{{ manufacturer_part.optimal_seller.ncnr }}</td>
+                                    <td id="sourcing-mp-{{ manufacturer_part.id }}-used">{{ manufacturer_part.optimal_seller.preferred }}</td>
+                                    <td id="sourcing-mp-{{ manufacturer_part.id }}-date">{{ manufacturer_part.optimal_seller.date }}</td>
```

Comment:
I believe putting the raw date here will print the date in UTC time, not local time. Is that intentional? Theres an example in this same file around line 480 that shows how we're using javascript to convert a django date into the browser's local time.

Reply:
Indeed. Since I'm in western Europe, UTC time is close to the timezone, and the UTC date will only be wrong during the night ;) But I assume this is not the same for US citizens, so yes, I need to change that.

Reply:
I had a look at the function around line 480, but I need to find something else, as the dates come from `for each` iteration loops. I have poor knowledge of javascript so I'll have to make some research to find how to do that.

Reply:
Here it is : commit d02a7b33b04ee4e79a6447bc8374e6aac0c3a3cf should fix the timezone issue. Could you review it @mpkasp ?